pkg/intel/metadata/fit/table.go

```diff
@@ -22,6 +23,31 @@ func (table Table) GetEntries(firmware []byte) (result []Entry) {
 	return
 }
 
+// Print prints the fit table in a tabular form
+func (table Table) String() string {
+	var s strings.Builder
+	// PrintFit prints the Firmware Interface Table in a tabular human readable form.
+	s.WriteString("Firmware Interface Table\n")
+	s.WriteString("------------------------\n")
+	s.WriteString(fmt.Sprintf("%-25s | ", "Type") +
```

Comment:
Just in case: it is also possible to do `fmt.Fprintrf(&s, ...)` instead of `s.WriteString(fmt.Sprintf(...))` if you would like this way.

Reply:
I will try that

Reply:
Done